### PR TITLE
fixed rubocop offenses, 11 remained

### PR DESCRIPTION
**Pull request by elioncho**

### CODE REVIEW

**Comment by elioncho** [comment]
[Add rubocop](https://trello.com/c/ECxaKHw1/19-add-rubocop)

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -12,16 +12,11 @@ def show
...
-    @game.user = current_user
-    respond_to do |format|
-      if @game.save
-        format.html { redirect_to @game, notice: 'El juego ha iniciado.' }
-        format.json { render :show, status: :created, location: @game }
-      else
-        format.html { render :new }
-        format.json { render json: @game.errors, status: :unprocessable_entity }
-      end
+    @game = Game.new(user: current_user)
```
Comment (EdgarOrtegaRamirez) [inline app/controllers/games_controller.rb]: Please use `current_user.games.new`

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -7,38 +7,23 @@ def flash_alerts
...
-      content_tag(:span, "X", class: "red")
-    end
-  end
-
   def friendly_move_message(move)
     "fila: #{(move.row + 65).chr}, columna: #{move.column + 1} - #{message.action}"
   end
 
   def ships_coordinates
-    ship_coordinates ||= @game.all_ships_coordinates
```
Comment (EdgarOrtegaRamirez) [inline app/helpers/application_helper.rb]: This `ship_coordinates` variable does not seem to be defined in the file, please remove all unused variables.

File: app/models/game.rb [hunk 3]
```diff
@@ -37,23 +35,21 @@ def set_ships
...
   def all_ships_coordinates
-    coordinates = []
     placed_ships.each do |placed_ship|
       (0...placed_ship.ship.size).each do |increment|
-        if placed_ship.orientation == Orientation::HORIZONTAL
-          coordinates << { row: placed_ship.row, column: placed_ship.column + increment  }
-        else
-          coordinates << { row: placed_ship.row + increment, column: placed_ship.column  }
-        end
+        coordinates << if placed_ship.orientation == GameShip::HORIZONTAL
```
Comment (EdgarOrtegaRamirez) [inline app/models/game.rb]: `coordinates` does not seem to be defined, please fix this.

File: app/models/game_ship.rb [hunk 4]
```diff
@@ -1,23 +1,25 @@
 class GameShip < ApplicationRecord
-  validates :orientation, inclusion: { in: [Orientation::HORIZONTAL, Orientation::VERTICAL] }
+  HORIZONTAL = 0
```
Comment (EdgarOrtegaRamirez) [inline app/models/game_ship.rb]: @elioncho please add a prefix to these constants `ORIENTATION_HORIZONTAL`.

File: app/models/game.rb [hunk 5]
```diff
@@ -17,7 +17,6 @@ def belongs_to?(user)
 
   def set_ships
     Ship.all.each do |ship|
-      puts "Setting #{ship.name} with size: #{ship.size}"
       loop do
         orientation = [Orientation::HORIZONTAL, Orientation::VERTICAL].sample
```
Comment (EdgarOrtegaRamirez) [inline app/models/game.rb]: Since you remove the `Orientation` concern, how is this working?

File: app/models/game.rb [hunk 6]
```diff
@@ -86,14 +81,16 @@ def can_place_ship_at?(size, row, column, orientation)
     return true if placed_ships.empty?
     placed_ships.each do |placed_ship|
       new_ship_coordinates = if orientation == Orientation::HORIZONTAL
```
Comment (EdgarOrtegaRamirez) [inline app/models/game.rb]: Since you remove the `Orientation` concern, how is it working?

File: app/models/game_ship.rb [hunk 7]
```diff
@@ -1,23 +1,25 @@
...
+                       { row: row + increment, column: column  }
+                     end
     end
     coordinates
   end
 
   def hit!
     self.hits += 1
-    puts "#{self.ship.name} hits: #{self.hits}"
+    puts "#{ship.name} hits: #{self.hits}"
```
Comment (EdgarOrtegaRamirez) [inline app/models/game_ship.rb]: Remove `puts` and `p` please.

File: app/models/game.rb (hunk 3 above)
Comment (elioncho) [inline app/models/game.rb]: True, lo había quitado arriba y ahí sería una variable local dentro del loop.

File: app/models/move.rb [hunk 8]
```diff
@@ -9,16 +9,15 @@ class Move < ApplicationRecord
   validate :game_is_on_progress, on: :create
 
   belongs_to :game
-  belongs_to :game_ship
+  belongs_to :game_ship, optional: true
```
Comment (EdgarOrtegaRamirez) [inline app/models/move.rb]: 👍

**Pull request merged**